core/lexicon/en/setting.inc.php

```diff
@@ -84,9 +84,6 @@
 $_lang['setting_login_homepage_desc'] = 'Enter the ID of the document you want to send the user to after he/she has logged in. <strong>NOTE: Make sure the ID you enter belongs to an existing document, and that it has been published and is accessible by this user!</strong>';
 
 // system settings
-$_lang['setting_access_policies_version'] = 'Access Policy Schema Version';
-$_lang['setting_access_policies_version_desc'] = 'The version of the Access Policy system. DO NOT CHANGE.';
-
```

Comment:
I believe these should stay because the related system setting seems to be used during the setup/upgrade of MODX:

https://github.com/modxcms/revolution/blob/c44faaf70d96ad351d713e7d8cced20124cbeed4/setup/includes/upgrade.install.php#L161-L330

Reply:
Yes, but I deleted only the lexicon, the setting itself is not present in the system settings grid. https://github.com/modxcms/revolution/blob/3.x/_build/data/transport.core.system_settings.php

Reply:
That looks like it might be a remnant from the upgrade to v2.0.5. I believe we're not planning to support 3.0 upgrades from anything but 2.6/2.7, so that looks like code which could also be removed.

Reply:
I was wrong. This setting is specified during the upgrade. Now I will return the lexicon.

Reply:
@Mark-H Sent a commit before your comment :)
I think that all lexicons can be cleaned with a separate PR group. Let this lexicon remain for now.